wgpu-hal/src/gles/adapter.rs

```diff
@@ -328,11 +328,14 @@ impl super::Adapter {
             gl.get_parameter_i32(glow::MAX_VERTEX_UNIFORM_BLOCKS)
                 .min(gl.get_parameter_i32(glow::MAX_FRAGMENT_UNIFORM_BLOCKS)) as u32;
 
-        let max_compute_workgroups_per_dimension = gl
-            .get_parameter_indexed_i32(glow::MAX_COMPUTE_WORK_GROUP_COUNT, 0)
-            .min(gl.get_parameter_indexed_i32(glow::MAX_COMPUTE_WORK_GROUP_COUNT, 1))
-            .min(gl.get_parameter_indexed_i32(glow::MAX_COMPUTE_WORK_GROUP_COUNT, 2))
-            as u32;
+        let max_compute_workgroups_per_dimension = if cfg!(not(target_arch = "wasm32")) {
```

Comment:
I think this check should be for GLES version not WASM32.
Something like this:
```rust
let supports_work_group_params = ver >= (3, 1);
let max_compute_workgroups_per_dimension = if supports_work_group_params {...
...
} else {...};
```

Reply:
done!